src/tools/webd/src/connector.js

```diff
@@ -16,14 +16,24 @@ const encodePath = path =>
 
 const version = host => fetch(`${host}/version`).then(res => res.json());
 
-const getRoot = (host, query = "") =>
-  fetch(`${host}/kdb${query}`).then(res => res.json());
+const getRoot = (host, sessionId, query = "") =>
```

Comment:
Did this change the API?

Reply:
no, we have to pass back the same session id per user to elektrad to allow reusage of a kdb handle